solr/core/src/java/org/apache/solr/response/GeoJSONResponseWriter.java

```diff
@@ -292,11 +293,18 @@ else if(geo instanceof WriteableGeoJSON) {
     }
   }
 
+  @Deprecated
   @Override
   public void writeStartDocumentList(String name, 
-      long start, int size, long numFound, Float maxScore) throws IOException
+      long start, int size, long numFound, Float maxScore) throws IOException {
+    throw new UnsupportedOperationException();
```

Comment:
Why? We can't pick a default and still use this?

Reply:
This method Shouldn't be called at all for writing a response. In other ResponseWriters, I left the code untouched for support in case someone is extending the Writer. In this case, since it's an inner class, I decided to throw an exception to catch potential bugs (internal code calling this)